sample-code/spring-app/src/test/java/com/sap/ai/sdk/app/controllers/OrchestrationTest.java

```diff
@@ -114,8 +113,8 @@ void testMaskingAnonymization() {
     var maskingResult = result.getModuleResults().getInputMasking();
     assertThat(maskingResult.getMessage()).isNotEmpty();
     var data = (Map<String, Object>) maskingResult.getData();
-    var maskedMessage = ((List<Map<String, Object>>) data.get("masked_template")).get(0);
-    assertThat(maskedMessage.get("content"))
```

Comment:
Those tests should fail on main

Reply:
Please explain.

Reply:
They do fail on main, the orchestration service must have changed the response ([maybe relevant](https://sap-cpe.slack.com/archives/G01T43F30CU/p1732523396578289))
![image](https://github.com/user-attachments/assets/7dde25e7-7342-49da-badc-e9e4ec4b8d3e)

Reply:
The Slack link doesn't work for me, but yeah, looks like this was changed on server side. I'll create a PR to update the test assertion